packages/config/package.json

```diff
@@ -64,7 +64,7 @@
     "jest-canvas-mock": "2.5.0",
     "jest-environment-jsdom": "29.7.0",
     "merge-stream": "2.0.0",
-    "rollup": "4.21.2",
+    "rollup": ">=4.22.4",
```

Comment:
This would allow any future rollup version. We need to pin the rollup version to 4.22.4 here, or at least use a caret.

Reply:
Ah thanks for catching that!